src/applications/accredited-representative-portal/routes.jsx

```diff
@@ -1,27 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
+import { createRoutesWithSaveInProgress } from 'platform/forms/save-in-progress/helpers';
+
 import App from './containers/App';
-import LandingPage from './containers/LandingPage';
-import POARequestsPage from './containers/POARequestsPage';
-import SignedInLayoutWrapper from './containers/SignedInLayoutWrapper';
+import formConfig from './accreditation/21a/config/form';
 
-const routes = [
-  {
-    component: App,
-    childRoutes: [
-      {
-        path: '/',
-        component: LandingPage,
-      },
-      {
-        component: SignedInLayoutWrapper,
-        childRoutes: [
-          {
-            path: '/poa-requests',
-            component: POARequestsPage,
```

Comment:
was this route removed entirely?

Reply:
Yes this isn't part of the app now. The plan I believe is to remove this as dead code and maintain the option to retrieve it from git history.

Reply:
Copy that, thank you